app/consumer-democracy/app.go

```diff
@@ -521,6 +594,7 @@ func New(
 		feegrant.ModuleName,
 		paramstypes.ModuleName,
 		vestingtypes.ModuleName,
+		genutiltypes.ModuleName,
```

Comment:
Missing consensus in all of these

Reply:
The apps are for testing only. We don't use it anywhere in testing.

Reply:
Done